images/rhel/rhel8/ubi8-toolbox/Containerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
I think we should specify the exact version with a tag. For example:
```
FROM registry.access.redhat.com/ubi8/ubi:8.5
```

Also, the directory structure should probably be `images/rhel/rhel-8.5/Containerfile` to match the dist-git layout. ie., roughly each dist-git branch is a sub-directory.

Reply:
Changed directory structure and changed the source for the base imafge. Also changed the naming and version of the image.